GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -22,6 +22,11 @@ public class LoggingBroker : ILoggingBroker
 
         public void LogDebug(string message) =>
             this.logger.LogDebug(message);
+
+        public void LogWarning(string message) =>
+            this.logger.LogWarning(message);
```

Comment:
Remove LogWarning.  The scope of this PR is only to add LogCritical.   When we werge changes in from main, then LogWarning will be added as part of the incoming changes from main.